lib/constants.js

```diff
@@ -59,3 +61,4 @@ exports.IDENTITY_FN = val => val;
 
 exports.isWindows = platform === 'win32';
 exports.isMacos = platform === 'darwin';
+exports.isLinux = platform === 'linux';
```

Comment:
What about BSD?

Reply:
The problem exists for `inotify` only (AFAIK) which is strictly Linux